pkg/logentry/stages/util.go

```diff
@@ -64,6 +65,22 @@ func convertDateLayout(predef string, location *time.Location) parser {
 		}
 	case "Unix":
 		return func(t string) (time.Time, error) {
+			if strings.Count(t, ".") == 1 {
+				split := strings.Split(t, ".")
+				if len(split) != 2 {
+					return time.Time{}, fmt.Errorf("Can't split %v into two parts", t)
```

Comment:
```suggestion
					return time.Time{}, fmt.Errorf("can't split %v into two parts", t)
```

Not your fault, in go error should not be capitalized. :)